storage/src/test/scala/coop/rchain/storage/StorageActionsTests.scala

```diff
@@ -326,6 +374,12 @@ abstract class StorageActionsTests extends FlatSpec with Matchers with OptionVal
       getK(r1).results shouldNot contain theSameElementsAs getK(r2).results
       getK(r1).results shouldNot contain theSameElementsAs getK(r3).results
       getK(r2).results shouldNot contain theSameElementsAs getK(r3).results
+
+      store.withTxn(store.createTxnWrite()) { txn =>
+        store.removeJoin(txn, "ch1", List("ch1"))
```

Comment:
Are we doing this right?  Should there still be joins in the table at this point?